code/modules/clothing/suits/jobs.dm

```diff
@@ -168,7 +168,7 @@
 /obj/item/clothing/suit/hazardvest/worn_overlays(mutable_appearance/standing, isinhands, icon_file)
 	. = ..()
 	if(!isinhands)
-		. += emissive_appearance(icon_file, "[icon_state]-emissive", src, alpha = src.alpha)
+		. += emissive_appearance(standing, "[icon_state]-emissive", src, alpha = src.alpha)
```

Comment:
this uh, doesn't make any sense. why does a mutable appearance work as an icon arg?

Reply:
why change it at all?

Reply:
> Yep -- still works, after changing to `standing.icon` too. I'm not sure why it worked with just standing but that is a mystery I am too tired to delve into right now.

I don't trust byond